engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/required-tags.service.js

```diff
@@ -0,0 +1,77 @@
+
+/**
+ * @ngdoc service
+ * @name  Bastion.products.details.repositories.service:RequiredTags
+ *
+ * @description
+ *   Helper functions for repo requiredTags.
+ */
+
+angular
+    .module('Bastion.repositories')
+    .service('RequiredTags', function () {
```

Comment:
can ya add a unit test for this? look at https://github.com/Katello/katello/blob/master/engines/bastion_katello/test/products/details/repositories/checksum.service.test.js for example